.gitmodules

```diff
@@ -12,6 +12,7 @@
 	path = ghcide
 	# url = https://github.com/digital-asset/ghcide.git
 	# url = https://github.com/alanz/ghcide.git
-	url = https://github.com/wz1000/ghcide.git
+	# url = https://github.com/wz1000/ghcide.git
+	url = https://github.com/haskell/ghcide.git
```

Comment:
what is the impact of this change?

Reply:
Functionally, nothing. It's a vanity commit.

Reply:
I see!